packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/info.jsx

```diff
@@ -18,7 +18,7 @@ const MultipliersInfo = ({
     multiplier = 0,
     has_stop_loss,
     is_tooltip_relative,
-    should_show_tooltip,
+    is_should_show_tooltip,
```

Comment:
[1/6] I think `should_show_tooltip` is a good name indicating it is a `boolean` type